q2_diversity/_version.py

```diff
@@ -23,9 +23,9 @@ def get_keywords():
     # setup.py/versioneer.py will grep for the variable names, so they must
     # each be defined on a line of their own. _version.py will just call
     # get_keywords().
-    git_refnames = "$Format:%d$"
-    git_full = "$Format:%H$"
-    git_date = "$Format:%ci$"
+    git_refnames = " (tsneupload)"
+    git_full = "7d64dcdf1efff39a893f3a3ab132d159bcc25b19"
+    git_date = "2021-05-03 11:12:26 +0200"
```

Comment:
This shouldn't have changed as it's an internal detail to packaging for the plugin.